gobblin-audit/src/main/java/gobblin/audit/values/sink/FsAuditSink.java

```diff
@@ -70,7 +70,7 @@
 
   public FsAuditSink(Config config, ValueAuditRuntimeMetadata auditMetadata) throws IOException {
 
-    this.fs = FileSystem.get(URI.create(ConfigUtils.getString(config, ConfigurationKeys.FS_URI_KEY, ConfigurationKeys.LOCAL_FS_URI)), new Configuration());
+    this.fs = FileSystem.get(new Configuration());
```

Comment:
The path should be initialized first. Then:

 this.fs = this.auditDirPath.getFileSystem(new Configuration());